app/main/views/agreements.py

```diff
@@ -17,6 +17,17 @@
                                                   # much easier to just act as a filter
 ))
 
+OLD_COUNTERSIGNING_FLOW_FRAMEWORKS = [
+    'digital-outcomes-and-specialists-4',
+    'digital-outcomes-and-specialists-3',
+    'digital-outcomes-and-specialists-2',
+    'digital-outcomes-and-specialists',
+    'g-cloud-11',
+    'g-cloud-10',
+    'g-cloud-9',
+    'g-cloud-8',
+]
+
```

Comment:
Should we move the e-signature check i.e. https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/bba069b6e3d680ef106872a7119e03636a6aff17/app/main/helpers/frameworks.py#L478-L479 into utils so admin and supplier can share the logic?

Reply:
Hmm good question. In the admin it's a slightly different usage, as we show a different signature template for G-Cloud 7 and earlier. Having it be consistent across apps would be useful though...